node_modules/col-assets/lib/api.js

```diff
@@ -55,7 +55,8 @@ var UserConstants = require('col-users/lib/constants');
 var getAssetProfile = module.exports.getAssetProfile = function(ctx, id, incrementViews, callback) {
   var opts = {
     'incrementViews': incrementViews,
-    'includeWhiteboardElements': true
+    'includeWhiteboardElements': true,
+    'includeWhiteboardUsages': true
   };
```

Comment:
That makes sense. Unfortunately, showing the active whiteboards in which an asset is used isn't all that helpful as whiteboards are private and we wouldn't be able to send the user to the whiteboard. What would be useful is to include which exported whiteboards the asset is being used in, and I'd be happy with including that in the scope of this project. In terms of the logic associated to deciding whether a user can delete an asset, I'd prefer that to be consolidated in the back-end so we can update that logic without changing the UI.